core/templates/services/UpgradedServices.ts

```diff
@@ -1247,6 +1255,10 @@ export class UpgradedServices {
         upgradedServices['UrlInterpolationService'],
         upgradedServices['LoaderService'],
         upgradedServices['WindowRef']);
+    upgradedServices['ContentTranslationManagerService'] =
```

Comment:
@srijanreddy98 are these changes correct?

Reply:
It is not a problem since ContentTranslationLanguageService wasn't added to the servicesToProvide in App.ts. However, @shavavo do the unit tests fail because of not having this change?